### PR TITLE
Standardizing output of sql2txt with all dependencies addressed

### PR DESCRIPTION
**Pull request by namma**
Output of sql2txt is now in general/standard format. If a column is array, the output is "[..]". If a column is struct, the output is a json string. 

There are 5 dependencies on the old sql2txt. They have been updated wrt the new ouput format.
- getVectorQuantiles
- getFiveNumSummary
- 2 internal functions inside BinningHandler.java
- getVectorHistogram (using HiveSQ)

In which, getVectorHistogram is totally rewritten to avoid using sql2txt. 

All DDF unit tests are passed.

### CODE REVIEW

**Comment by khangich** [comment]
ok to test

**Comment by namma** [comment]
@mbbui @Huandao0812 @nhanitvn please review

**Comment by mbbui** [comment]
retest this please

File: core/src/main/java/io/ddf/analytics/AStatisticsSupporter.java [hunk 1]
```diff
@@ -329,7 +332,7 @@ public void setThirdQuantile(double mThird_quantile) {
 
     mLog.info("Column type: " + colType);
     List<Double> pValues = Arrays.asList(percentiles);
-    Pattern p1 = Pattern.compile("(big|small|tiny|int)");
+    Pattern p1 = Pattern.compile("(big|small|tiny|int|long)");
```
Comment (mbbui) [inline core/src/main/java/io/ddf/analytics/AStatisticsSupporter.java]: why not "bigint", "smallint" etc?

Comment (mbbui) [inline core/src/main/java/io/ddf/analytics/AStatisticsSupporter.java]: @nhanitvn @Huandao0812 do you know this?

Comment (namma) [inline core/src/main/java/io/ddf/analytics/AStatisticsSupporter.java]: I've made a test using the this code
'''
Pattern p1 = Pattern.compile("(big|small|tiny|int|long)");
System.out.println(p1.matcher("small").matches());
System.out.println(p1.matcher("smallint").matches());
System.out.println(p1.matcher("int").matches());
'''
The result is true, false, true, respectively. Thus, that line of code (Line 335) might not work correctly.

Comment (Huandao0812) [inline core/src/main/java/io/ddf/analytics/AStatisticsSupporter.java]: proper way to check for column type:
https://github.com/ddf-project/DDF/commit/cd446875b4a95e215e53dc385a7d96c8564fa305
@namma @mbbui

**Comment by namma** [comment]
I tested the new update from @Huandao0812 for getVectorQuantiles. It works. 
@khangich @mbbui please retest

**Comment by namma** [comment]
retest this please

**Comment by mbbui** [comment]
retest this please

**Comment by khangich** [comment]
ok to test

**Comment by mbbui** [comment]
retest this please

**Comment by mbbui** [comment]
retest this please

**Pull request merged**